ph5/clients/tests/test_ph5tostationxml.py

```diff
@@ -6,16 +6,18 @@
 import sys
```

Comment:
These unit test seems quite a bit different. Does it seem safe to change a large portion of code and unit test. It's hard to tell but it seems like some refactoring has occurred. If this code was refactored, do we want to do this in the same PR that is changing logic as well?

Reply:
All of the change in test_ph5tostationxml.py are because ph5tostationxml.py has been changed to not return empty response but raise error when there are errors for no response data or response table has no response filenames at all.
When the behaviors are changed from logging error to raising error the tests must be changed a lot.

Reply:
Thanks for this explanation.